packages/components/toggle-group/src/props.ts

```diff
@@ -48,12 +48,7 @@ export const toggleGroupProps = {
   props: {
     type: {
       type: String as PropType<ToggleGroupSingleProps['type'] | ToggleGroupMultipleProps['type']>,
-      validator: (value: string): boolean => {
-        if (!value)
-          console.error(`Missing prop \`type\` expected on \`${TOGGLE_GROUP_NAME}\``)
-
-        return true
-      },
+      required: true,
     },
```

Comment:
I think `Vue` doesn't call `validator` when the prop is not provided. So, I change it to `required`. What do you think?

Reply:
I agree with you